ddlite.py

```diff
@@ -1063,11 +1064,11 @@ def learn_weights(self, **kwargs):
     result = learn_elasticnet_logreg(F=F[train_cov, :], 
                                      LF=LF[train_cov, :],
                                      w0=w0, unreg=unreg, mu_seq=mu, **kwargs)
-    w_all, self._lf_marginals = result, None    
+    w_all, self._lf_w = result, None    
     if pipeline:
       zw = np.zeros_like(result[0])
       w_all = {k : np.concatenate((zw, v)) for k,v in result[1].iteritems()}
-      self._lf_marginals = result[2]
+      self._lf_w = result[2]
```

Comment:
Currently, **_lf_w** is just a list of marginals and **training()** is the id list of training data. They have the same length, but how to find matching between these two lists.